src/util.js

```diff
@@ -58,7 +60,8 @@ export function serializeForm(form) {
 		case "input":
 			switch(node.type) {
 			case "file":
-				throw new Error("`input[type=file]` fields are unsupported");
+				value = node.files[0];
```

Comment:
```suggestion
				// NB: due to `enctype` differentiation, files and
				//     `StringParams` should be mutually exclusive
				value = node.files[0];
```

also, what about `<input type="file" multiple>`; there'd be more then one file there, no?

Reply:
I of course did not test that 🙈 

I'm wondering if we could just use `value = node.files` here, since we iterate over `value` in a later step

Reply:
Based on a cursory check, it doesn't seem to work that way:

```html
    <form>
        <input type="file" name="attachments" multiple>
    </form>
    <script>
(function() {

document.querySelector("input").addEventListener("change", ev => {
    let field = ev.target;
    console.log(field.files);

    let params = new FormData();
    params.append(field.name, field.files);
    console.log(params, [...params.keys()], [...params.values()]);
});

}());
    </script>
```

Assigning `field.files` instead of `field.files[0]` there appears to result in stringification: `"object FileList]"` 🤷

Reply:
I would hope that iterating and invoking #append individually would work.

that is what happens here anyway: https://github.com/FND/hijax-form/blob/master/src/util.js#L89, so I would hope that we wouldn't have to change too much in the implementation